include/mata/nfa/builder.hh

```diff
@@ -45,10 +46,12 @@ Nfa create_sigma_star_nfa(Alphabet* alphabet = new OnTheFlyAlphabet{});
 /** Loads an automaton from Parsed object */
 // TODO this function should the same thing as the one taking IntermediateAut or be deleted
 Nfa construct(const Mata::Parser::ParsedSection& parsec, Alphabet* alphabet, StringToStateMap* state_map = nullptr);
+//construct does not cary much info, what is better? In the comment, don't know what Parsed object is, overall I don't get what this function does.
 
```

Comment:
I think `construct` is OK, `construct_from` does not bring any information. One can check easily what `ParsedSection` is.

Reply:
I personally like naming conventions of pandas and other Python libraries that would use `read_section` or `from_parsed_section`. 

Btw. There Is information in `from` 😉

Reply:
Construct is saying that the function is constructing something, but deosn't know what. Kind of useless name in fact. At least construct_nfa, no? But isn't that function actually doing the parsing? In that case it could be parse_nfa?  (although it may be weird to parse a parsed section, but that is perhaps another problem)

Then, I don't like that we use those synonyms - make, create, construct, without any clear system. That could be another argument for just dropping them when they are not necessary.

Reply:
I don't have a strong opinion on that. But where is the line? Make is bad, get is good?

Reply:
Wasted energy, I am for purging them. Sorry for that I am probably getting into a fundamentalist mode while writing these comments :). Instead of spending energy on creating a system, that will be artificial and so endlessly discussed, not respected, enforced, reexplained, lets concentrate on figuring out each individual name so that it is informative. For me, get create make construct says almost nothing, except that the function is creating and returning some thing, which I probably easily know anyway from the context, from how it is used, from return type. 
Allahu Akbar!

Reply:
> I would suggest to adapt system from other libraries, that would name these kinds of fucntions as from_<FORMAT> and to_<FORMAT>. So this could be Builder::from_parsed_section(), or Builder::nfa_from_parsed_section().

I think this could work for Mata very well, too. We already have `OrdVector::to_vector()`, so we could have `Nfa::Nfa::to_mata()` and `Nfa::Nfa::from(ParsedSection)`, etc.